en/02_Developer_Guides/02_Controllers/01_Introduction.md

```diff
@@ -84,12 +80,11 @@ Action methods can return one of four things:
 1. an `HTTPResponse`. This can either be a new response or `$this->getResponse()`.
 1. `$this` or `$this->customise()`. This will render the controller using the appropriate template and set the rendered result as the body for the current `HTTPResponse`.
 
-[hint]
-
-- returning `$this` is the equivalent of returning an empty array.
-- returning `$this->customise()` is the equivalent of returning an array with data.
-
-[/hint]
+> [!TIP]
+> There are a couple of things to note here:
+>
+> - returning `$this` is the equivalent of returning an empty array.
+> - returning `$this->customise()` is the equivalent of returning an array with data.
```

Comment:
Added text above the list because we need a blank link before the list to satisfy [MD032](https://github.com/DavidAnson/markdownlint/blob/main/doc/md032.md), and it would look weird with just a blank spot there.